frontend/src/components/LandingPage/TeamCard/TeamCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  Card, /*CardImg, CardText,*/ CardBody,
+  CardTitle, CardSubtitle, /*Button*/
+} from 'reactstrap';
+import Image from 'react-bootstrap/Image';
+import './TeamCard.css'
+
+const TeamCard = (props) => {
+  console.log(props.headshot)
+  return (
+    <span>
+    <Card>
+        <br />
+        <Image className = "headshot" src = {props.headshot} alt = {props.name}  roundedCircle/>
+        <CardBody className="text-center">
+          <CardTitle tag="h5">{props.name}</CardTitle>
+          <CardSubtitle tag="h6" className="mb-2 text-muted">{props.line1}<br />{props.line2}<br />{props.line3}</CardSubtitle>
+          {/*<CardText>Some quick example text to build on the card title and make up the bulk of the card's content.</CardText>*/}
```

Comment:
Can you remove this comment line as it is not necessary anymore.

Reply:
done